roles/core/dhcp_server/molecule/default/molecule.yml

```diff
@@ -0,0 +1,75 @@
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: management1
+    groups:
+      - mg_managements
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+    docker_networks:
+      - name: bbnet
+        ipam_config:
+          - subnet: '10.10.0.0/16'
```

Comment:
Could you use 10.11.0.0/16 instead?

10.10.0.0/16 is widely used in the examples, I already implemented public (and private) environments with this subnet in [bluebanquise-vagrant](https://github.com/bluebanquise/bluebanquise-vagrant). Once the Vagrant env is up on a dev host, it is not possible to run this molecule test on the same dev host because the IP address will conflict between Vagrant and Docker.

Reply:
Corrected in commit : 879b3ae

Interesting the  [bluebanquise-vagrant](https://github.com/bluebanquise/bluebanquise-vagrant) project. Maybe not the place to discuss about it, but, why you do not use molecule ?
All the steps in this script shell [bootstrap.sh](https://github.com/bluebanquise/bluebanquise-vagrant/blob/master/tiny-hpc-cluster/bootstrap.sh) can be achieved with ansible tasks (in a `prepare.yml `file for example)
Also the [VagrantFile ](https://github.com/bluebanquise/bluebanquise-vagrant/blob/master/tiny-hpc-cluster/Vagrantfile) can be generated from a `molecule.yml` file

Reply:
To be honest I started this project one week before starting to play with molecule and did not think of it.

I'm using this project to quickly setup several environments with respect to BB's ansible inventory structure, for development and for testing. I may use it later for trainings as well, to quickly spawn environments for students. I need it to be compatible with libvirt and Virtualbox (and more recently Hyper-V which is still WIP), I don't know if molecule can create the relevant VagrantFile for all these backends.

If you want to discuss this further, feel free to open a new [discussion](https://github.com/bluebanquise/bluebanquise/discussions) 😀